ansible/ansible.cfg

```diff
@@ -155,7 +155,9 @@ callback_whitelist = profile_tasks
 # wanting to use, for example, IP information from one group of servers
 # without having to talk to them in the same playbook run to get their
 # current IP information.
-fact_caching = memory
+fact_caching = jsonfile
+fact_caching_connection = ~/.ansible/cache
+fact_caching_timeout = 86400
```

Comment:
we are really using those data. the only data we need is the kernel version and distribution, I'll decrease it to 10 minutes.